src/EditorFeatures/Test2/EditorServicesTest2.vbproj

```diff
@@ -80,6 +79,7 @@
     <PackageReference Include="Microsoft.VisualStudio.Text.UI" Version="$(MicrosoftVisualStudioTextUIVersion)" />
     <PackageReference Include="Microsoft.VisualStudio.Text.UI.Wpf" Version="$(MicrosoftVisualStudioTextUIWpfVersion)" />
     <PackageReference Include="Microsoft.VisualStudio.Validation" Version="$(MicrosoftVisualStudioValidationVersion)" />
+    <PackageReference Include="System.Threading.Tasks.Dataflow" Version="$(SystemThreadingTasksDataflowVersion)" />
```

Comment:
Nit: if this is implied by Microsoft.VisualStudio.Composition and everything works by deleting this line, I'm OK with not stating it. Our projects today over-specify a lot of package versions which makes things get annoying sometimes.